terraform/modules/front_door/alarms.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_cloudwatch_metric_alarm" "unhealthy_hosts_count" {
```

Comment:
I was quite unsure about datapoints/periods/threshold here. For now I've just put it so that we'll know very quickly if there's any unhealthiness at all (as that seems useful to me!), but lmk if you think we want to monitor over a longer period. E.g. could check for 3 datapoints in 5x 1 minute periods.

Reply:
Yeah think I agree. I believe this is based of healthchecks failing which are likely to happen either because the app/host is genuinely unhealthy, is under so much load the endpoint is not responding / timing out or the AZ has gone down 😱

The healthcheck has criteria itself too before the app/host is deemed unhealthy (see example below), so imo I think it's ok to flag this right away 😊

```
health_check {
    healthy_threshold   = "3"
    interval            = "30"
    protocol            = "HTTP"
    matcher             = "204"
    timeout             = "3"
    path                = "/health"
    unhealthy_threshold = "2"
  }
```

Reply:
It's not necessarily clear that we'd want to know about all unhealthy hosts, for the case where one becomes unhealthy and is replaced and then everything continues fine.

I'm happy to stick with this and tweak it later though.